common.gypi

```diff
@@ -341,6 +342,7 @@
             'xcode_settings': {
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
               'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++0x',  # -std=gnu++0x
+              'OTHER_CPLUSPLUSFLAGS' : ['-stdlib=libc++'],
```

Comment:
I think linking to libc++ makes it a semver-major change because it affects compiled add-ons.